.github/workflows/docs-quality-checker.yml

```diff
@@ -0,0 +1,37 @@
+name: Check Markdown files quality
+
+on:
```

Comment:
shouldn't this be something like:

```
on:
  pull_request:
    branches: [master]
```

since we only want to run the jobs on pull requests to master?